lma-ai-stack/README_UBAC_MEETING_SHARING.md

```diff
@@ -0,0 +1,45 @@
+# User Based Access Control: Meeting Sharing Feature
+Starting in version 0.2.5, LMA users can share meetings that they own with other users. On the Meeting List page of the LMA web UI, users can now see the meetings they own as well as meetings that have been shared with them.
+
+In the following illustration, we see meeting List seen by User 1 (kkdaws@amazon.com):
+1. The first meeting in the list is owned by a different user, but because it has been shared with User 1, it shows up in User1's meeting list. You can see the owner under Owner Email and the list of users the meeting has been shared with under Shared With.
+2. The second and third meetings in the list are owned by User 1, and shared with User 2 (kkdaws+1@amazon.com)
+
+![Share Meeting View User 1](./images/meeting-sharing-view-user-1.png)
+
+Here we see the meeting List seen by User 2 (kkdaws+1@amazon.com):
+1. The first meeting in the list is owned is the same as the first meeting in User1's list above - this is the meeting that is owned by User 2, and shared with User 1.
```

Comment:
typo:  The first meeting in the list **is owned is the same** as the first meeting in User1's list above